services/controllerhost/consumer.go

```diff
@@ -363,6 +363,85 @@ func fetchClassifyOpenCGExtents(context *Context, dstUUID string, cgUUID string,
 	return
 }
 
+// listConsummableExtents returns the list of extents for the destination that
+// are in open or sealed state and can be consumed by the given CG.
```

Comment:
also make it clear that the returned dstExtents *aren't already consumed by the CG*